### ISSUE TITLE
The position of the animation in the animation controls does not match to the equivalent value in the field of the TimeLine controls

### ISSUE THREAD

**Issue by JosueReis** [godotengine/godot#42393]
**Godot version:**
3.2.3

**OS/device including version:**
GForce GTX 750 Ti, driver: 27.21.14.5638, GLES2

**Issue description:**
When using the animation position selector in Animation Controls, the changed value should comply with that specified in Timeline Controls

**Steps to reproduce:**
Click on one of the two arrows in the "animation position" field in animation controls to select a value. The value that appears is different from the one specified in the field of the timeline controls

**Minimal reproduction project:**

[teste.zip](https://github.com/godotengine/godot/files/5295640/teste.zip)

**Comment by Calinou** [comment]
Can you take a screenshot of the issue, please? I don't understand exactly which fields you're referring to.

**Comment by JosueReis** [comment]
Ok, my fellow.
![Godot](https://user-images.githubusercontent.com/61919659/111881005-4c64ca80-898d-11eb-8c45-1653c305e931.png)

**Comment by JosueReis** [comment]
Grateful for the reply, but only now have I seen another way to achieve
the expected result, because although I can't use the selector, I can simply move the bar below it to get the expected result.

**Comment by akien-mga** [comment]
Yes the normal way to seek to a specific position to insert a new value is to click directly on the timeline. The SpinBox lets you seek to the intermediate values so you can see e.g. the interpolation between two discrete values.
